backend/api/MQTT/MqttService.cs

```diff
@@ -0,0 +1,186 @@
+﻿using System.Text.Json;
+using Api.MQTT.Events;
+using Api.MQTT.MessageModels;
+using Api.Utilities;
+using MQTTnet;
+using MQTTnet.Client.Connecting;
+using MQTTnet.Client.Disconnecting;
+using MQTTnet.Client.Options;
+using MQTTnet.Client.Receiving;
+using MQTTnet.Extensions.ManagedClient;
+
+namespace Api.MQTT
```

Comment:
I believe abbreviations above three letters should have lower case e.g. `Mqtt`. This has been used in the class name, should it also be used for the folder and namespace names?

Reply:
Agreed